aws-lc-rs/src/rand.rs

```diff
@@ -166,40 +166,45 @@ pub fn fill(dest: &mut [u8]) -> Result<(), error::Unspecified> {
 mod tests {
     use crate::rand;
     use std::array::IntoIter;
+    use std::ops::Range;
 
     use crate::rand::{generate, SecureRandom, SystemRandom};
 
+    const RAND_ARY_SIZE: usize = 191;
+    const EXPECTED_MEAN_RANGE: Range<f64> = 101f64..155f64;
+    const EXPECTED_VARIANCE_LB: f64 = 8f64;
```

Comment:
where do these values come from?

Reply:
I can't really defend these specific numbers, but the range checked for is ~5 standard deviations above/below the mean.

Expected Mean: E(X) = (255-0)/2 = 127.5
Expected Variance: Var(X) = ((256^2)-1)/12 = 5461.25
Standard Deviation of n-bytes: (Var(X)/n)^(1/2) = (5461.25/191)^(1/2) = ~5.34

(What's called variance here is actually the standard deviation.)